zipline-loader/src/commonMain/kotlin/app/cash/zipline/loader/ZiplineLoader.kt

```diff
@@ -552,6 +559,8 @@ class ZiplineLoader internal constructor(
     eventListener: EventListener,
     manifestUrl: String,
   ): LoadedManifest {
+    // Option 1: wrap L563 - L577 in try catch, and collect `retrySignal` in this function
+    // perform retry on every emission from `retrySignal`
```

Comment:
the focus/decision is what to be retried upon: networking itself or loading zipline included